ocsp/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "x509-ocsp"
```

Comment:
It'd probably be good if the directory name matched the crate name (i.e. `x509-ocsp`)

I should probably rename the `x509` directory to `x509-cert` for consistency as well.

Reply:
I changed the ocsp folder.